pypdf/annotations/_base.py

```diff
@@ -1,27 +1,124 @@
 from abc import ABC
+from typing import Any, Callable, Dict, Iterable, Optional
 
 from ..constants import AnnotationFlag
-from ..generic import NameObject, NumberObject
-from ..generic._data_structures import DictionaryObject
+from ..generic import (
+    ArrayObject,
+    BooleanObject,
+    DictionaryObject,
+    FloatObject,
+    NameObject,
+    NumberObject,
+    RectangleObject,
+    TextStringObject,
+)
 
+DEFAULT_FLAGS = AnnotationFlag(0)
+### !!!!!!! 0 is not actually no flags but the default value !!!!!
```

Comment:
It is both - 0 means no flags are active and that is the default

Reply:
exclamations were for finding my comments more easily.
**NO_**FLAGS is not clear for me for example whe you are looking at bit `Hidden` That's why I prefer the word DEFAULT